spec/spec_helper.rb

```diff
@@ -41,6 +47,8 @@
 Capybara.javascript_driver = :poltergeist # This is slower
 Capybara.default_max_wait_time = ENV['TRAVIS'] ? 30 : 15
 
+ActiveJob::Base.queue_adapter = :inline
```

Comment:
Because the test suite assumes synchronous processing and the new default is `:async`?